### ISSUE TITLE
[Feature]: Key broadcasting to multiple panes

### ISSUE THREAD

**Issue by ToontjeM** [wavetermdev/waveterm#1587]
### Feature description

The one thing i am desperately missing in Wave is key broadcasting to multiple panes. Imagine you have 4 panes open en in all four panes you need to log in to a different server and install some software. Currently you have to type the same commands in all four panes. Ideally Wave should be able to broadcast keyboard input to all those panes so you only have to type the commands once.

### Implementation Suggestion

I wish i could help you here. But look at how iTerm2 and Warp have implemented it and you know what i mean.

### Anything else?

Basically iTerm's CMD-Shift-I or Warp's, well, also CMD-Shift-I.

**Comment by esimkowitz** [comment]
this is included in 0.11, which should ship later today
